doc/modules/feature_selection.rst

```diff
@@ -70,16 +70,16 @@ as objects that implement the ``transform`` method:
    selection with a configurable strategy. This allows to select the best
    univariate selection strategy with hyper-parameter search estimator.
 
-For instance, we can perform a :math:`\chi^2` test to the samples
-to retrieve only the two best features as follows:
+For instance, we can perform a F-test to the samples to retrieve only the two
+best features as follows:
```

Comment:
```suggestion
For instance, we can use a F-test to retrieve the two
best features for a dataset as follows:
```
I like this better/sound more "english" to my ears. I used "dataset" as the collective noun for "samples", couldn't think of a better one.

Reply:
agreed :)